image.yaml

```
@@ -3,6 +3,11 @@
 # streams.
 include: image-base.yaml
 
+# Deploy via container now
+# https://github.com/coreos/fedora-coreos-tracker/issues/1823
+deploy-via-container: true
+container-imgref: "ostree-remote-registry:fedora:quay.io/fedora/fedora-coreos:next-devel"
```

Considering https://github.com/coreos/fedora-coreos-config/pull/3355 should we be using this instead:

```suggestion
container-imgref: "ostree-remote-registry:fedora:registry:quay.io/fedora/fedora-coreos:next-devel"
```

---

actually [it turns out](https://coreos.github.io/rpm-ostree/container/#url-format-for-ostree-native-containers) `ostree-remote-registry:fedora:` is equivalent to `ostree-remote-image:fedora:registry:` so the shorter form is actually better.